src/view/restore-wallet.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import SeedInputView from '../component/seed-input';
+
+//
+// Restore Wallet View
+//
+
+const RestoreWalletView = ({ store, nav, wallet }) => (
+  <SeedInputView
+    title="Let's restore your wallet"
+    copy={store.seedVerifyCopy}
+    navBack={nav.goSeed}
+    seedInputs={store.wallet.seedVerify}
+    indexes={store.seedVerifyIndexes}
+    setSeedVerify={wallet.setSeedVerify}
+    checkSeed={wallet.checkSeed}
+  />
```

Comment:
A view inside of another view 😃 We should probably rename this to a just `SeedInput`.

![recursion](https://media.giphy.com/media/ZLWnbaMlDjzGg/giphy.gif)